core/templates/pages/community-dashboard-page/community-dashboard-page.directive.html

```diff
@@ -22,10 +22,10 @@
         </span>
       </div>
       <div class="oppia-contributor-username-container" data-tab-name="username">
-        <span ng-show="!$ctrl.username">Guest</span>
```

Comment:
This would still be better conceptually.

Reply:
Done!